web/components/njwds-extended/VerticalStepIndicator.tsx

```diff
@@ -3,12 +3,12 @@ import { useMountEffect, useOnWindowResize } from "@/lib/utils/helpers";
 
 interface Props {
   number: number;
-  order?: number;
+  visualIndex?: number;
```

Comment:
why optional?

Reply:
just in case we decide to reuse the step indicator without the sections, no reason for it not to be optional

Reply:
now I'm thinking more about that, is there any reason to have both of these props at all? we could just have a single prop representing "this is the number to show on the stepper" and the caller can decide whether that's the "step number" or the "visual index", and it would simplify this component's interface

Reply:
and we could replace `openStepNumbers` with an `isOpen` boolean or something instead, and similarly let the caller do that logical calculation, so that we don't need to have the list of numbers and `props.number` in this component